packages/solana-actions/src/createPostResponse.ts

```diff
@@ -29,7 +29,7 @@ export interface CreateActionPostResponseArgs<
   TransactionType = Transaction | VersionedTransaction,
 > {
   /** POST response fields per the Solana Actions spec. */
-  fields: Omit<ActionPostResponse, "transaction"> & {
+  fields: Omit<TransactionResponse, "transaction"> & {
```

Comment:
this works for now since the `createPostResponse` does not yet support the recently added optional transactions and external links are not supported at all in the current version of `@solana/actions`

either way, this will also require updating the spec package inside the `@solana/actions` package.
I just opened an issue to track it and will complete the update in a different PR: https://github.com/solana-developers/solana-actions/issues/34

Reply:
agreed, just wanted to address existing build errors

thanks for creating the issue